lib/prima_auth0_ex/application.ex

```diff
@@ -5,11 +5,15 @@ defmodule PrimaAuth0Ex.Application do
 
   require Logger
 
+  alias PrimaAuth0Ex.Config
   alias PrimaAuth0Ex.Telemetry
   alias PrimaAuth0Ex.{JwksStrategy, TokenProvider}
 
   def start(_type, _args) do
-    log_configuration_errors()
+    unless client_configured?() or clients_configured?() or server_configured?() do
```

Comment:
```suggestion
    unless Config.default_client() or Config.client() or Config.server() do
```

Do this work?

Reply:
I think it should...?

Reply:
Nope, `nil` is not falsy

Reply:
in fact it is: https://hexdocs.pm/elixir/1.12.3/Kernel.html#module-truthy-and-falsy-values

Reply:
Then I don't understand [this error](https://drone-1.prima.it/primait/auth0_ex/542/1/11) :confounded:

Reply:
`or` requires the arguments to be booleans, if you use `||` it should work

```
iex(1)> nil || true
true
iex(2)> nil or true
** (BadBooleanError) expected a boolean on left-side of "or", got: nil
    iex:2: (file)
```

Reply:
or is the "strictly boolean" operator, while || uses also nil

Reply:
https://hexdocs.pm/elixir/1.12.3/Kernel.html#or/2

Reply:
@MaeIsBad technically only the left one needs to be boolean. `true or 2` works :vomiting_face:

Reply:
TIL! Thanks :smile_cat: